tests/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "diagram",
+  "name": "tests",
```

Comment:
Should this be "e2e tests"?

Reply:
This is auto-generated so I'll leave it.